libraries/ms-common/inc/chip_id.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
```

Comment:
Since users of the module will look at the header for how to use it, you should add a few comments here to document what the module is. In particular, mention that it's for accessing the hardware chip ID (point them to the datasheet like you did in the stm32 impl file) and mention that it's constant for every controller board. You could also mention how it's emulated in x86 if you want.